### PR TITLE
feat(svg-icon): add kv-custom-dashboard and kv-back icons

### PR DESCRIPTION
**Pull request by djcouto**

### CODE REVIEW

**Review by pedrorebelocosta — APPROVED** [review]

File: packages/ui-components/src/assets/svg-symbols.svg [hunk 1]
```diff
@@ -1034,6 +1034,15 @@
 		<path d="M4 20h16v2H4v-2zM4 2h16v2H4V2zm9 7h3l-4-4-4 4h3v6H8l4 4 4-4h-3V9z"/>
 	</symbol>
 
+	<symbol id="kv-dashboard-custom"
```
Comment (ricardogoncalves89) [inline packages/ui-components/src/assets/svg-symbols.svg:1037]: Missing add to the list (`packages/react-ui-components/src/stories/foundation/SvgIcons/SvgIcons-list.tsx`) to be visible in the SVG story

**Review by ricardogoncalves89 — CHANGES_REQUESTED** [review]

File: packages/ui-components/src/assets/svg-symbols.svg (hunk 1 above)
Comment (djcouto) [reply packages/ui-components/src/assets/svg-symbols.svg:1037]: Fixed 👍

Comment (djcouto) [reply packages/ui-components/src/assets/svg-symbols.svg:1037]: Also done

**Review by miguelpinto98 — APPROVED** [review]

**Review by ricardogoncalves89 — APPROVED** [review]
Pls build the packages to update the documentation

**Review by ricardogoncalves89 — APPROVED** [review]

**Pull request merged**